modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/view/UserOrderableServiceVo.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+
+package org.eclipse.xpanse.modules.models.servicetemplate.view;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+import java.util.UUID;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.eclipse.xpanse.modules.models.service.common.enums.Category;
+import org.eclipse.xpanse.modules.models.service.common.enums.Csp;
+import org.eclipse.xpanse.modules.models.servicetemplate.Billing;
+import org.eclipse.xpanse.modules.models.servicetemplate.DeployVariable;
+import org.eclipse.xpanse.modules.models.servicetemplate.Flavor;
+import org.eclipse.xpanse.modules.models.servicetemplate.Region;
+import org.springframework.hateoas.RepresentationModel;
+
+/**
+ * The view model for the end-user to query the available services.
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class UserOrderableServiceVo extends RepresentationModel<UserOrderableServiceVo> {
+
+    @NotNull
+    @Schema(description = "The id of the available service.")
```

Comment:
available service. -> orderable service everywhere

Reply:
Whether to modify the filed in the related  API and Swagger description？
![image](https://github.com/eclipse-xpanse/xpanse/assets/121531362/c2488f88-a605-4ab5-a436-c0d0d80d2089)

Reply:
@baixinsui yes, we change this too.

Reply:
fixed all.